src/design-system/components/TextShadow/TextShadow.tsx

```diff
@@ -56,8 +57,14 @@ export const TextShadow = ({
       }),
     [blur, color, disabled, enableInLightMode, inferredTextColor, isAnimatedText, isDarkMode, shadowOpacity, x, y]
   );
-
-  if (IS_TEST || (IS_ANDROID && !enableOnAndroid)) return children;
+  if (IS_TEST || (IS_ANDROID && !enableOnAndroid)) {
```

Comment:
We need to make sure to apply containerStyle in this case. This wasn't a regression from this PR, but included the fix here since it is pretty simple.